src/__tests__/lib/containers/entity/page/title_bar/EntityPageTitleBar.test.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
```

Comment:
Other tests that are "renamed" in diff were moved from `src/_tests_` to `src/__tests__` (double underscore)